tests/test_register_schema.py

```diff
@@ -0,0 +1,130 @@
+from contextlib import redirect_stderr
+import json
+import jsonschema
+import logging
+import pytest
+import io
+
+from jupyter_telemetry.eventlog import EventLog
+
+
+def test_register_invalid():
+    """
+    Test registering invalid schemas fails
+    """
+    el = EventLog()
+    with pytest.raises(jsonschema.SchemaError):
+        el.register_schema({
+            # Totally invalid
+            'properties': True
+        })
+
+    with pytest.raises(ValueError):
+        el.register_schema({
+            'properties': {}
+        })
+
+    with pytest.raises(ValueError):
+        el.register_schema({
+            '$id': 'something',
+            '$version': 1,
+            'properties': {
+                'timestamp': {
+                    'type': 'string'
+                }
+            }
+        })
+
```

Comment:
Should we add a test that fails because a dunder key of `properties` is set in the given schema?

Reply:
Added!